dependencies.yml

```diff
@@ -3,7 +3,7 @@
 #     If its classes are exposed in Javadoc, update offline links as well.
 #
 boms:
-- com.linecorp.armeria:armeria-bom:0.72.0
+- com.linecorp.armeria:armeria-bom:0.72.1-SNAPSHOT
```

Comment:
Will remove `-SNAPSHOT` once https://github.com/line/armeria/pull/1377 is published.